assets/js/components/Beneficiaries.vue

```diff
@@ -271,17 +299,31 @@ export default Vue.extend({
       /*
        *
        */
-      bentered.append("title").text(
+      bentered.attr("title",
         (d) => d.map(
           (d_) => d_.fm + ":\t" + this.format(d_.value)
-        ).join("\n")
+        ).join("<br>")
       )
+
+      let tip = d3.tip()
+        .attr('class', 'd3-tip benef')
+        .html(function(d){return d})
+        .direction('n').offset([-20, 0]);
+
+      chart.call(tip)
+
+      bentered.on('mouseover', function(){
+        tip.show(d3.select(this).attr('title') + "<br> <span>Click to filter by beneficiary state</span>")
```

Comment:
is this how the author suggests you use the library, in his extremely simple, extremely obvious, extremely direct example?